src/frontend/src/pages/FinancePage/EditReimbursementRequest/EditReimbursementRequest.tsx

```diff
@@ -28,11 +30,19 @@ const EditReimbursementRequestPage: React.FC = () => {
 
   const onSubmit = async (data: ReimbursementRequestDataSubmission): Promise<string> => {
```

Comment:
this is actually already try, catched where onSubmit is called so we don't need to do it here as well